config/300-bitbucketsource.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    eventing.knative.dev/source: "true"
+  name: bitbucketsources.sources.eventing.knative.dev
```

Comment:
perhaps just `bitbucket.sources.eventing.knative.dev`

Reply:
I agree it looks better as you propose. I basically copied the all the other sources.
I think that first part of the name (bitbucketsources) need to match spec.names.plural. Otherwise it complaints.
I'll leave it as is for now, unless there are further objections.

Reply:
+ @grantr

Reply:
`bitbucket.sources.eventing.knative.dev` does look nicer. I worry that in practice, people won't use the fully qualified name, and then commands look like this:

```
kubectl get bitbuckets
kubectl get githubs
kubectl get kubernetesevents
```

which looks weird to me.

I think I prefer `bitbucketsources.sources.eventing.knative.dev` until someone commits to updating (and documenting) the naming convention and changing the names of existing sources. Otherwise, `Bitbucket` will remain the oddball because nobody thought to change `GitHubSource` too.